cromwell-drs-localizer/src/main/scala/drs/localizer/downloaders/AccessUrlDownloader.scala

```diff
@@ -0,0 +1,28 @@
+package drs.localizer.downloaders
+
+import cats.effect.{ExitCode, IO}
+import cloud.nio.impl.drs.AccessUrl
+import com.typesafe.scalalogging.StrictLogging
+
+import scala.sys.process.{Process, ProcessLogger}
+
+case class AccessUrlDownloader(accessUrl: AccessUrl, downloadLoc: String) extends Downloader with StrictLogging {
+
+  def generateDownloadScript(): String = {
+    val signedUrl = accessUrl.url
+    // TODO headers, requester pays, refinements to retry strategy for certain 4xx statuses
+    s"""mkdir -p $$(dirname '$downloadLoc') && curl --location --retry 3 --retry-connrefused --retry-delay 10 --fail --output '$downloadLoc' '$signedUrl'"""
```

Comment:
ToL: The change for requester pays will require changes to the request sent to martha, not changes to this line here. But ToL I think it's fine to lump the TODOs together.